Revert "Revert "Update documentation on FreeMarker template support"" (#2876)
wso2/docs-ei: en/micro-integrator/docs/references/mediators/payloadFactory-Mediator.md

## Syntax

!!! Info

>     The template-type attribute is released as a product update on <b>07/01/2021</b>. If you don't already have this update, you can [get the latest updates](https://updates.docs.wso2.com/en/latest/updates/overview/) now.

Reviewer: ```suggestion
    The template-type attribute is released as a product update on <b>07/01/2021</b>. If you do not have this update, you can [get the latest updates](https://updates.docs.wso2.com/en/latest/updates/overview/) now.
```
